content/middlewares.md

```diff
@@ -20,6 +20,10 @@ Nest middleware are, by default, equivalent to [express](https://expressjs.com/e
 
 You implement custom Nest middleware in either a function, or in a class with an `@Injectable()` decorator. The class should implement the `NestMiddleware` interface, while the function does not have any special requirements. Let's start by implementing a simple middleware feature using the class method.
 
+>  warning **Warning** `Express` and `fastify` treat middleware differently and have different syntaxes for middleware, for more information read [here](https://docs.nestjs.com/techniques/performance#middleware).
```

Comment:
Same here https://github.com/nestjs/docs.nestjs.com/pull/2589/files#r1096776454 @jmcdo29 - not sure if "treat" word makes the most sense here

Reply:
We could use the word "handle" instead. I think that conveys the meaning better with less room for interpretation

Reply:
```suggestion
>  warning **Warning** `Express` and `fastify` handle middleware differently and have different syntaxes for middleware, for more information read [here](https://docs.nestjs.com/techniques/performance#middleware).
```

ok ?